### PR TITLE
Add tests for paginations

### LINKED ISSUE #5393: Add pagination tests for connection fields with filtering and sorting
In PR #5363 we add many pagination tests for connection fields with filtering and sorting. We need to add similar tests for queries:
1. [ ] service_accounts
1. [ ] customers
1. [ ] staff_users
1. [ ] orders
1. [ ] draft_orders

### PR DESCRIPTION
**Pull request by koradon**
I want to merge this change because...
It adds missing tests for queries with pagination.

fixes #5393

TODO:
- [x] add test for orders queries with pagination and sortBy

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [x] Privileged queries and mutations are guarded by proper permission checks
* [x] Database queries are optimized and the number of queries is constant
* [x] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by fowczarek** [comment]
For pagination tests, we have module `tests\api\pagination`. Please move tests related to pagination with `filtering` or `sorting` to this module.

File: tests/api/pagination/test_account.py [hunk 1]
```diff
@@ -1,8 +1,249 @@
...
+        ({"search": "example.com"}, 2),
+        ({"search": "Alice"}, 1),
+        ({"search": "Kowalski"}, 1),
+        ({"search": "John"}, 1),  # default_shipping_address__first_name
+        ({"search": "Doe"}, 1),  # default_shipping_address__last_name
+        ({"search": "wroc"}, 1),  # default_shipping_address__city
+        ({"search": "pl"}, 2),  # default_shipping_address__country, email
+    ],
+)
+def test_query_customer_members_pagination_with_filter_search(
```
Comment (fowczarek) [inline tests/api/pagination/test_account.py:134]: This test checks if filtering working correctly. We need to prepare data with giving as at least 3 results and we need to make sure that the first page has the first 2 of 3 results.
We can prepare only 1 parameter per `CustomerFilter` for example one for "search", "moneySpent" ...

File: tests/api/pagination/test_order.py [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+    staff_api_client.user.user_permissions.add(permission_manage_orders)
+    response = staff_api_client.post_graphql(
+        QUERY_DRAFT_ORDERS_WITH_PAGINATION, variables
+    )
+    content = get_graphql_content(response)
+    assert content["data"]["draftOrders"]["totalCount"] == 1
+
+
+# Currently sorting is not working properly for orders.
+# We should uncomment this test after fixing
```
Comment (fowczarek) [inline tests/api/pagination/test_order.py:528]: We should add an issue or PR id here.

Comment (koradon) [reply tests/api/pagination/test_order.py]: Done

File: tests/api/pagination/test_account.py (hunk 1 above)
Comment (koradon) [reply tests/api/pagination/test_account.py:262]: Done

**Pull request merged**